src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -23,4 +23,35 @@ public int divide(int a, int b) throws IllegalArgumentException {
         }
         return a / b;
     }
+
+    // Intentional errors added below
```

Comment:
5. **Inconsistent Naming:**
   - The `Subtract()` method starts with an uppercase letter, which is inconsistent with the naming conventions for Java methods. Method names should start with a lowercase letter.